code/modules/mod/mod_theme.dm

```diff
@@ -1159,6 +1159,88 @@
+		This modsuit model is one of rare fruits created by their joint operations, mashing scrapped designs with super soldier enhancements. \
+		Already light, when powered on, this modsuits injects the wearer seemlessly with muscle-enhancing supplements, while adding piston strenght \
+		to their legs. Combination of these mechanisms is very energy draining - but results in next to no speed reduction for the wearer.\
+		Over the years, many a rich person, including Nanostrasen officials with premium subscriptions, had their life or genes rescued thanks to \
+		unrivaled speed of this suit model. Equally as many, however, mysteriously dissapeared in the flash of these white suits after they forgot \
+		to pay off said subscryptions in due time or posted unfavourable opinions on Intrudyne's gene-modding tech and ethics. "
+	default_skin = "interdyne"
+	armor_type = /datum/armor/mod_theme_interdyne
+	resistance_flags = FIRE_PROOF|ACID_PROOF
+	max_heat_protection_temperature = FIRE_SUIT_MAX_TEMP_PROTECT
+	charge_drain = DEFAULT_CHARGE_DRAIN * 2.2
+	slowdown_inactive = 0.5
+	slowdown_active = 0.1
+	inbuilt_modules = list(/obj/item/mod/module/quick_carry/advanced, /obj/item/mod/module/organ_thrower)
+	allowed_suit_storage = list(
+		/obj/item/healthanalyzer,
+		/obj/item/reagent_containers/dropper,
+		/obj/item/reagent_containers/cup/beaker,
+		/obj/item/reagent_containers/cup/bottle,
+		/obj/item/reagent_containers/cup/tube,
+		/obj/item/reagent_containers/hypospray,
+		/obj/item/reagent_containers/pill,
+		/obj/item/reagent_containers/syringe,
+		/obj/item/stack/medical,
+		/obj/item/sensor_device,
+		/obj/item/storage/pill_bottle,
+		/obj/item/storage/bag/chemistry,
+		/obj/item/storage/bag/bio,
+		/obj/item/melee/baton/telescopic,
+		/obj/item/ammo_box,
+		/obj/item/ammo_casing,
+		/obj/item/restraints/handcuffs,
+		/obj/item/assembly/flash,
+		/obj/item/melee/baton,
+		/obj/item/melee/energy/sword,
+		/obj/item/shield/energy,
+	)
+	skins = list(
+		"interdyne" = list(
+			HELMET_FLAGS = list(
+				UNSEALED_LAYER = NECK_LAYER,
+				UNSEALED_CLOTHING = SNUG_FIT,
+				SEALED_CLOTHING = THICKMATERIAL|STOPSPRESSUREDAMAGE|BLOCK_GAS_SMOKE_EFFECT|HEADINTERNALS,
+				UNSEALED_INVISIBILITY = HIDEFACIALHAIR,
+				SEALED_INVISIBILITY = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEHAIR|HIDESNOUT,
+				SEALED_COVER = HEADCOVERSMOUTH|HEADCOVERSEYES|PEPPERPROOF,
+			),
+			CHESTPLATE_FLAGS = list(
+				UNSEALED_CLOTHING = THICKMATERIAL,
+				SEALED_CLOTHING = STOPSPRESSUREDAMAGE,
+				SEALED_INVISIBILITY = HIDEJUMPSUIT,
+			),
+			GAUNTLETS_FLAGS = list(
+				UNSEALED_CLOTHING = THICKMATERIAL,
+				SEALED_CLOTHING = STOPSPRESSUREDAMAGE,
+				CAN_OVERSLOT = TRUE,
+			),
+			BOOTS_FLAGS = list(
+				UNSEALED_CLOTHING = THICKMATERIAL,
+				SEALED_CLOTHING = STOPSPRESSUREDAMAGE,
+				CAN_OVERSLOT = TRUE,
+			),
+		),
+	)
+
+/datum/armor/mod_theme_interdyne
```

Comment:
That's pretty damn weak for an antagonist piece of armor. To contrast, the pirate coats the Jolly Roger gets are some of the best armor in the game, and they're spaceproof by default. Maybe give them the armor module syndies get?

Reply:
buffed the speed and armour now.